src/backlight/metrics/pl.py

```diff
@@ -20,6 +22,20 @@ def _divide(a: float, b: float) -> float:
     return a / b if b != 0.0 else 0.0
 
 
+def calc_sharp(
+    positions: Positions, principal: float, freq: pd.Timedelta = pd.Timedelta("60min")
```

Comment:
Is "60min" a standard frequency for computing sharpe? I think it's application specific, if so let's not put a default

Reply:
Okay! Thanks.